.env

```diff
@@ -20,5 +20,5 @@ REDIS_PORT=6379
 REDIS_USERNAME="default"
 # REDIS_PASSWORD=""
 
-AIRTABLE_BASE_ID=
-AIRTABLE_API_KEY=
+AIRTABLE_API_KEY=keyChbhVlLmxHNsor
```

Comment:
Hey @abtheo you should create a `.env.local` for this and never commit to public repo 😅

Reply:
> Hey @abtheo you should create a `.env.local` for this and never commit to public repo 😅

Oh crap I forgot I added this, it's all over my branch history 😅 removed in latest commit but gonna have to regenerate the keys, sorry!

Reply:
You can use `git rebase -i` to remove the key from history entirely.

Reply:
I'll rebase and force push a new fix on this 😅